tests/test_sir.py

```diff
@@ -1,42 +1,46 @@
 # pylint: disable=C0114
 # pylint: disable=C0116
 # pylint: disable=R0124
+# pylint: disable=R0201
 # pylint: disable=C0121
 "Test exponential convergence"
-# from models.sir import SIR
 import numpy as np
+import pytest
+
 from models.sir import SIR
 
+DEFAULT_PARAMS = [0.95, 0.38] # Default parameters from WHO
+EALING_IC = [341555, 445, 0]  # Ealing initial conditions
 
-def test_sixdays():
-    # Check that six days prediction is accurate with validated test case
-    p = [0.95, 0.38]  # Default parameters from WHO
-    nw0 = [341555, 445, 0]  # Ealing initial conditions
-    t = 6  # 7 days
-    my_sir = SIR()
-    my_sir.set_params(p, nw0)
-    my_sir.solve(t, t + 1)
-    removed_end = my_sir.fetch()[-1, -1]
+class TestSir:
```

Comment:
If the SIR Model tests are already grouped by the module, why do we need to group them on the class?

Reply:
I add a class here because we might add later unittests for the SIR functions too.
So, we could have "TestSir" and one for unit tests